src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -113,7 +187,7 @@ const PersonalDetails = (() => {
                 if (populate) {
                     const should_update_value = /select|text/i.test(element_key.type);
                     if (has_label) {
-                        CommonFunctions.getElementById(`row_${element_id}`).setVisibility(1);
+                        CommonFunctions.getElementById(`row_lbl_${element_id}`).setVisibility(1);
```

Comment:
how was it working before?

Reply:
template naming was not consistent. I've made a change to prefix all labels with `row_lbl`.